common/src/stack/command/stack/commands/add/pallet/__init__.py

```diff
@@ -326,13 +326,18 @@ def run(self, params, args):
 		paths_to_args = {data['exploded_path']: data['canonical_arg'] for data in pallet_args.values()}
 
 		# we have everything we need, copy the pallet to the fs, add it to the db, and maybe patch it
+		self.beginOutput()
 		for pallet in flatten(pallet_infos.values()):
 			self.copy(stacki_pallet_dir, pallet, clean)
 			self.write_pallet_xml(stacki_pallet_dir, pallet)
 			if updatedb:
 				self.update_db(pallet, paths_to_args[pallet.pallet_root])
 			if stacki_pallet_dir == '/export/stack/pallets':
 				self.patch_pallet(pallet)
+			self.addOutput(pallet.name, (pallet.version, pallet.release, pallet.arch, pallet.distro_family))
+		self.endOutput(header=['pallet', 'version', 'release', 'arch', 'os'])
 
 		# Clear the old packages
 		self._exec('systemctl start ludicrous-cleaner'.split())
+
+
```

Comment:
You got some extra newlines at the end of the file. Other than that, looks good.